### ISSUE TITLE
Generating par2 while uplading? "Generate on the fly"

### ISSUE THREAD

**Issue by scriptzteam** [animetosho/Nyuu#58]
Would be cool if somehow we can generate par2 while uploading, is it possible?

**Comment by animetosho** [comment]
Yes, it's theoretically possible. In fact, doing so was the plan with ParPar integration (if I ever get around to it) - Nyuu essentially sends read data to both the server and ParPar at the same time, effectively running the two in parallel and reducing overall disk I/O.

**Comment by JavascriptHiFi** [comment]
hi

have you find a way to get it to work?

**Comment by animetosho** [comment]
It hasn't been implemented yet. This issue should get updated when it is.

**Comment by JavascriptHiFi** [comment]
please please make my dream a reality xD

**Comment by Ravencentric** [comment]
is there any update on parpar's integration into nyuu?

**Comment by animetosho** [comment]
No update at this stage.

**Comment by animetosho** [comment]
Unbelievable right?

**Comment by Ravencentric** [comment]
> How come this is almost a most have for NZB files and it hasn't been implemented in over 4 years?

Being rude certainly won't manifest the feature out of thin air.

On a related note, while I wait for this feature, I wrote a python wrapper around nyuu and parpar to basically do this [here](https://github.com/Ravencentric/juicenet-cli). Not perfect but it works for my usage.

**Comment by Oleekae** [comment]
I am reopening this issue to ask if there is any hope that this can be done at some point? I am aware that it is a huge task in terms of code and optimisation and that it is clearly not easy. 

Thank you for continuing to develop nyuu, an excellent tool!

**Comment by animetosho** [comment]
The best answer I can give you is that I haven't been considering working on this lately.

**Comment by Oleekae** [comment]
Completely honest answer, appreciate your directness.
